starter-code/pizza.js

```diff
@@ -1 +1,81 @@
+'use strict'
 // Write your Pizza Builder JavaScript in this file.
+
+
+var $greenPep = $('div#pizza section.green-pepper'); 
```

Comment:
Why are you nesting so many selectors? You want to take everything with .green-pepper class, it is enough with that.